js/controller.js

```diff
@@ -823,6 +844,13 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       this.mb.publish(OO.EVENTS.DISCOVERY_API.SEND_DISPLAY_EVENT, eventData);
     },
 
+    sendVideoQualityChangeEvent: function(selectedContentData) {
+      this.state.videoQualityOptions.selectedBitrate = {
+        "id": selectedContentData.id
+      };
+      this.mb.publish(OO.EVENTS.SET_TARGET_BITRATE, selectedContentData.selectedBitrate.id);
```

Comment:
@mistermendez, doesn't work this way. Need to change selectedContentData.selectedBitrate.id to selectedContentData.id.